ci-operator/config/openshift/cluster-api/openshift-cluster-api-release-1.0.yaml

```diff
@@ -0,0 +1,59 @@
+base_images:
+  base:
+    name: "4.9"
+    namespace: ocp
+    tag: base
+  openshift_release_golang-1.16:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+binary_build_commands: make manager-core
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
+images:
+- build_args:
+  - name: builder_image
+    value: docker.io/library/golang:1.16.8
+  - name: ARCH
+    value: amd64
+  - name: ldflags
+  from: base
+  inputs:
+    openshift_release_golang-1.16:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.16
+  to: cluster-api
+promotion:
+  namespace: hypershift
+  tag: v1.0.0
+releases:
```

Comment:
Not clear to me how any of these are used

Reply:
Actually neither is clear to me, dropped it.